### PR TITLE
Modify DG

### PR DESCRIPTION
**Pull request by jowhee3011**
**Modifying Use Case**
__Assign__
Case 1e, 1f: Not implemented yet, is currently an alpha bug

**Adding Manual Testing**
Changed Editw Error message to be similar to edit error message

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -288,109 +286,155 @@ Priorities:
...
-7. Bridal Boss confirms that the vendor has been added.
-
+1. Wedding Organizer adds a contact using the add command.
    Use case ends.
 
 **Extensions:**
 
-- **4a.** Wedding Organizer cancels the operation.
-    - **4a1.** Bridal Boss returns to the wedding details view.
+- **1a.** Name, phone, email, address prefixes not inputed
```
Comment (VedJoshi) [inline docs/DeveloperGuide.md:322]: very detailed

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -288,109 +286,155 @@ Priorities:
...
+| `*`      | Wedding organizer          | upload important documents (e.g., contracts)       | access them quickly during planning                                           |
+| `*`      | Wedding organizer          | send automated reminders to vendors and clients    | ensure they stay informed of upcoming deadlines                               |
+| `*`      | Wedding organizer          | set up recurring tasks for common preparations     | avoid manually creating the same tasks for each event                         |
 
 ### Use Cases
 
 (For all use cases below, the **System** is *Bridal Boss* and the **Actor** is the *Wedding Organizer*.)
 
-#### **Use Case: Add a New Vendor to a Wedding**
+#### **Use Case: Add a Contact**
```
Comment (VedJoshi) [inline docs/DeveloperGuide.md:313]: should we refer them to the user guide for the command?

**Review by VedJoshi — COMMENTED** [review]
LGTM  minor enhancements might be good

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (jowhee3011) [reply docs/DeveloperGuide.md:313]: howw hahah

Comment (VedJoshi) [reply docs/DeveloperGuide.md:313]: > howw hahah, bc in tht case everyth can be referred to the UG too

can link it at the top saying refer to all commands in the user guide?

Comment (jowhee3011) [reply docs/DeveloperGuide.md:313]: okies done!

**Review by SherniceSng — APPROVED** [review]
LGTM.

**Pull request merged**